common/pb/event_notify_user.proto

```diff
@@ -5,6 +5,7 @@ message Event_NotifyUser {
     enum NotificationType {
         PROMOTED = 1;
         WARNING = 2;
+        IDLEWARNING = 3;
```

Comment:
Yeah, Ill add an "UNKNOWN" enum value at the 0 location and add a client side switch statement so future clients will be able to show a message about the client being out of date.  Thanks.